pages/establishment/dashboard/views/index.jsx

```diff
@@ -147,6 +147,14 @@ const Index = ({
                 <Link page="establishment.asru" asruUser="spocs" label={ <Snippet>pages.edit</Snippet> } />
               }
             </dd>
+
+            {
+              establishment.sharedKey &&
+                <Fragment>
+                  <dt><Snippet>sharedKey</Snippet></dt>
+                  <dd>{establishment.sharedKey}</dd>
+                </Fragment>
+            }
```

Comment:
Should this be visible to _everyone_ who has visibility of the establishment?

Reply:
Nope, just ASRU, I'll fix.

Reply:
Can it be fixed in a way that reads the permissions from permissions rather than baking this particular permission into the UI?